tests/bootstrap.php

```diff
@@ -1,29 +0,0 @@
-<?php
```

Comment:
No. We are loading `vendor/bootstrap.php` as the bootstrap file in the phpunit configuration. The memory limit is there too.
Why we sat here an monolog Logger, I don't remeber.